GUI/Main.cs

```diff
@@ -979,11 +978,11 @@ private void exportModListToolStripMenuItem_Click(object sender, EventArgs e)
 
         private void selectKSPInstallMenuItem_Click(object sender, EventArgs e)
         {
-            Main.Instance.Manager.ClearAutoStart();
-            var old_instance = Main.Instance.CurrentInstance;
+            Instance.Manager.ClearAutoStart();
+            var old_instance = Instance.CurrentInstance;
             var result = new ChooseKSPInstance().ShowDialog();
-            if (!Equals(old_instance, Main.Instance.CurrentInstance))
-                Main.Instance.CurrentInstanceUpdated();
+            if (result == DialogResult.OK && !Equals(old_instance, Instance.CurrentInstance))
```

Comment:
Not with the current code. On a cancel the instance would not of been changed so the !Equals call would be false.